activerecord/test/models/topic.rb

```diff
@@ -99,7 +99,7 @@ def default_written_on
     end
 
     def destroy_children
-      self.class.where("parent_id = #{id}").delete_all
+      self.class.delete_by("parent_id = #{id}")
```

Comment:
I am not sure why we have used string interpolation here. `"parent_id = #{id}"` can be replaced with `parent_id: id`

Reply:
Yeah, that does seem odd. Does git blame or the Topic using tests say anything? While it doesn't matter in tests, this also technically is vulnerable to SQL injections so at the very least it should use `= ?`. But let's just switch it to the hash as you mention.

Reply:
@kaspth Changes seem to be very old can be changed to `parent_id: id`. I have done that here.